xmppserver/src/main/webapp/server-locale.jsp

```diff
@@ -261,6 +261,17 @@
                     <label for="zh_CN">Simplified Chinese (zh_CN)</label>
                 </td>
             </tr>
+             <tr>
+                <td>
+                    <input type="radio" name="localeCode" value="fa_IR" <%= ("fa_IR".equals(locale.toString()) ? "checked" : "") %> id="fa_IR" />
+                </td>
+                <td>
```

Comment:
I _think_ you need to use `fa` or `fa_IR` everywhere (here, and in other files) for this to work. You've been consistently using `fa` so I think you want to use that here, too.